ksqldb-functional-tests/src/test/resources/query-validation-tests/row-offset-and-partition.json

```diff
@@ -19,13 +19,14 @@
         "sources": [
           {"name": "OUTPUT", "type": "stream", "schema": "ID INT KEY, ROWPARTITION STRING, ROWOFFSET STRING"}
         ]
-      }
+      },
+      "comments": "Should be removed after rowpartition.rowoffset.enabled feature flag is removed"
```

Comment:
nit: it's nice to have "comments" above "name", else the comments are easily missed.